src/style.css

```
@@ -0,0 +1,79 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+
+  --color-white: #fff;
+  --color-silver: #e1e7ed;
+  --color-gray: #3d4e61;
+  --color-dark: rgba(61, 78, 97, 0.1);
+}
+
+.form1,
+.form2 {
+  position: relative;
+  margin-top: 20px;
+}
```

you did great with `.form__input` after recommendation, let's do the same with other elements. 
Common `.form` and modifier `.form--small`
Common `.form__image` and modifier `.form__image--small`
```suggestion
.form {
  position: relative;
  margin-top: 20px;
}
```